unit-tests/func/MUR/l500/test-auto-set-options.cpp

```diff
@@ -0,0 +1,35 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
I suggest `test-dependencies.cpp`.
It can probably be grouped with the next test

Reply:
We narrow it down to 3 exe, take another look.
I think it's OK now.

If you want all in 1 exe let me know